library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -443,8 +443,40 @@ public int markRowDeleted(long... ids) {
      * @param ids the IDs of the downloads to remove
      * @return the number of downloads actually removed
      */
-    public int remove(long... ids) {
-        return markRowDeleted(ids);
+    public int removeDownloads(long... ids) {
```

Comment:
Both public with different javadoc and one redirects to the other. Should be removed.